tasks/pg_migrate.rake

```diff
@@ -0,0 +1,46 @@
+require "sqlite3"
+require "pg"
+require './myapp'
+
+namespace :one_offs do
+  task :pg_migrate do
+    Message.destroy_all
+    Follower.destroy_all
+    User.destroy_all
```

Comment:
I'm not really a fan of dropping our old data
Can you elaborate what you have discussed in person @duckth @fredpetersen ?

I see that we do a select statement and maps data, but after destroy_all?

Could we not save the return values of it and use that to migrate data?
https://apidock.com/rails/ActiveRecord/Relation/destroy_all